example/addLocatorExample/addLocator_usage_example_spec.js

```diff
@@ -0,0 +1,9 @@
+describe('angular material checkboxes page', function() {
+  it('should click on checkbox', function() {
+    browser.get('https://material.angular.io/components/checkbox/overview');
+
+    element(by.checkboxText('Check me!')).click();
+
+    expect($('.mat-checkbox-input').getAttribute('aria-checked')).toEqual('true');
```

Comment:
We faced this question when testing checkboxes with material: is `aria-checked` the proper way to test a checkbox in e2e?

Reply:
Hi, @glepretre ! Actually, you can also use isSelected() function. In this case you will have:
```
expect($('.mat-checkbox-input').isSelected()).toBe(true);
```
But, I think that it depends on component structure. Here we have material component and as for me both methods works good.